models/Element/Service.php

```diff
@@ -1263,6 +1263,23 @@ function (Concrete $object, Data $fieldDefinition, $property, $currentValue) {
         return $theCopy;
     }
 
+    /**
+     * @template T
+     * @param T $properties
+     * @return T
+     *
+     * @return mixed
```

Comment:
Remove the second return
```suggestion
     * @template T
     *
     * @param T $properties
     *
     * @return T
```

Reply:
thank you! i have just noticed after the commit but you were faster 😄